app/src/main/java/org/p2p/wallet/infrastructure/network/NetworkModule.kt

```diff
@@ -48,6 +48,11 @@ object NetworkModule : InjectionModule {
     const val DEFAULT_CONNECT_TIMEOUT_SECONDS = 30L
     const val DEFAULT_READ_TIMEOUT_SECONDS = 30L
 
+    enum class MoonpayRetrofitQualifier {
```

Comment:
I suggest to move enum classes to separate file

Reply:
Also maybe we will separate network module, for example move Moonpay to separate module

Reply:
makes sense, but i would leave it here for the developer to know which qualifer to use when he wants to inject retrofit and stumbled upon NetworkModule